MAUI/DateTimePicker/date-time-picker-mode.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Date Time picker mode in .NET MAUI Date Time Picker control | Syncfusion
+description: Learn about date time picker mode in Syncfusion .NET MAUI Date Time Picker (SfDateTimePicker) control and its basic features.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# Date Time picker mode in .NET MAUI Date Time Picker (SfDateTimePicker)
+The Date Time picker mode is specified in the picker property enumeration, which is used to show the date time picker based on the modes. It provides three modes such as `Default`, `Dialog` and `RelativeDialog`. The default picker mode is `Default` in the [SfDateTimePicker].
+
+
+## Dialog mode
+
+The dialog mode is used to show the date time picker in a pop up by setting the [Mode] property to [`Dialog`] in [SfDateTimePicker].
```

Comment:
The dialog mode is used to display the date time picker in a popup by setting the [Mode] property to [`Dialog`] in [SfDateTimePicker].

Reply:
Content modified.